src/test/language-services/utils.ts

```diff
@@ -328,14 +334,14 @@ export function dumpNodeToTraceFile(node: PQP.Language.Ast.INode, filePath: stri
     File.writeFileSync(filePath, asJson);
 }
 
-const DefaultAnalysisOptions: AnalysisOptions = {};
+const DefaultAnalysisOptions: Configuration = {};
```

Comment:
If you keep the name "Configuration" then renamed this const.

Reply:
renamed back to AnalsisOptions